pkg/codegen/internal/test/testdata/external-resource-schema/python/pulumi_example/cat.py

```diff
@@ -0,0 +1,84 @@
+# coding=utf-8
+# *** WARNING: this file was generated by test. ***
+# *** Do not edit by hand unless you're certain you know what you are doing! ***
+
+import warnings
+import pulumi
+import pulumi.runtime
+from typing import Any, Mapping, Optional, Sequence, Union
+from . import _utilities, _tables
+from ._inputs import *
+from pulumi_random import RandomPet
```

Comment:
`RandomPet` doesn't appear to be used in this file, so not sure why it's being included in the imports (this may be a preexisting "issue"). Not that big of a deal as it doesn't really cause any harm being imported.

Reply:
It looks like this issue already existed. We could probably tighten up the import logic, but I'm also inclined to leave it alone for now since it doesn't break anything.

Reply:
Leaving alone sounds fine to me.